### PR TITLE
Asset List Page redesign

### PR DESCRIPTION
**Pull request by Space-zee**
https://app.clickup.com/t/860q7cpru

### CODE REVIEW

File: frontend/app/features/assets/controllers/AdminAssetsListController.tsx [hunk 1]
```diff
@@ -4,16 +4,34 @@ import {useAtomValue, useSetAtom} from "jotai";
...
   const assetsCombined = useAtomValue(assetsModel.$assetsCombinedFiltered)
+  const [activeTab, setActiveTab] = useState("all");
+  const [sortedAssets, setSortedAssets] = useState(assetsCombined);
 
   useEffect(() => {
     onPageLoad()
   }, [])
 
+  useEffect(() => {
+    if(assetsCombined){
```
Comment (ruslanpalagin) [inline frontend/app/features/assets/controllers/AdminAssetsListController.tsx:23]: is that possible to use model like I proposed (see in pm) ?

File: frontend/app/features/assets/models/assetsListModel.ts [hunk 2]
```diff
@@ -74,16 +78,42 @@ const withoutFilter = () => true
 const onlyAssetsThatAreVisibleToUser = (asset: { status: number }) =>
   [AssetStatuses.Upcoming, AssetStatuses.Active, AssetStatuses.SoldOut, AssetStatuses.Confirmed].includes(asset.status)
 
-const sortAssetsRule = (prev: AssetInfoCombinedV2, next: AssetInfoCombinedV2): -1 | 0 | 1 => {
+const sortAssetsRuleByOrder = (prev: AssetInfoCombinedV2, next: AssetInfoCombinedV2): -1 | 0 | 1 => {
   return prev.computed.uiOrder < next.computed.uiOrder ? -1 : 1;
 }
+const sortAssetsRuleByCollected = (prev: AssetInfoCombinedV2, next: AssetInfoCombinedV2) => {
+  const collectedNext = (Number(ethers.utils.formatEther(next.sellProgress.maxTotalSupply)) - Number(ethers.utils.formatEther(next.sellProgress.leftToBuy))) / Number(ethers.utils.formatEther(next.sellProgress.maxTotalSupply)) * 100;
```
Comment (ruslanpalagin) [inline frontend/app/features/assets/models/assetsListModel.ts:85]: in the assetV2
there is a "computed" object which is reserved to keep UI & helpers variables. 
It's better to calculate such numbers there and just compare those here in the sort function.

File: frontend/app/features/assets/views/AssetList/AssetListItem/index.tsx [hunk 3]
```diff
@@ -31,19 +29,17 @@ const AssetListItemBoxView = ({
...
                                 assetAddress,
                                 saleDate,
-                                tokenPrice
+                                tokenPrice,
+                                location
                               }: Props) => {
-  const shortenSubTitle = subTitle.substring(0, 50).concat('...');
-  const shortenTitle = title.length < 15 ? title : title.substring(0, 15).concat('...');
-
+  const totalPropertyPrice = tokensTotal * tokenPrice;
```
Comment (ruslanpalagin) [inline frontend/app/features/assets/views/AssetList/AssetListItem/index.tsx:41]: must be in model:
assetV2.computed.totalPropertyPrice 
or something like that.

File: frontend/app/features/assets/views/AssetList/AssetListMainItem/index.tsx [hunk 4]
```diff
@@ -0,0 +1,132 @@
...
+                                    tokensLeft,
+                                    tokensTotal,
+                                    assetAddress,
+                                    saleDate,
+                                    tokenPrice,
+                                    location,
+                                    rentStartDate
+                                  }: Props) => {
+  const totalPropertyPrice = tokensTotal * tokenPrice;
+  const collected = (tokensTotal - tokensLeft) / tokensTotal * 100;
```
Comment (ruslanpalagin) [inline frontend/app/features/assets/views/AssetList/AssetListMainItem/index.tsx:39]: should be in model

**Review by ruslanpalagin — CHANGES_REQUESTED** [review]

**Comment by ruslanpalagin** [comment]
Story looks very good actually. Thank you!

File: frontend/app/features/assets/views/AssetList/AssetListItem/index.tsx (hunk 3 above)
Comment (Space-zee) [reply frontend/app/features/assets/views/AssetList/AssetListItem/index.tsx:41]: +

File: frontend/app/features/assets/views/AssetList/AssetListMainItem/index.tsx (hunk 4 above)
Comment (Space-zee) [reply frontend/app/features/assets/views/AssetList/AssetListMainItem/index.tsx:39]: +

File: frontend/app/features/assets/models/assetsListModel.ts (hunk 2 above)
Comment (Space-zee) [reply frontend/app/features/assets/models/assetsListModel.ts:85]: +

**Pull request merged**